### DISCUSSION TITLE
upload animated model from blender

### DISCUSSION THREAD

**Discussion by Markusfenix535** [google/model-viewer#4178] [Q&A]
Someone explain me the configuration or the method to export an animated model from blender and make it work in a web page.

**Comment by WadeMorrison** [comment]
Basically, the process is like any other animation > GLB/GLTF export. You just need to ensure that when you animate the object in Blender, you have the 'Dope Sheet' window open, and have it set to "Action Editor" instead of "Dope Sheet". Add your keyframes, and then from there you can re-name the animation name if wanted, and then click the "Push Down" button to save it as a new strip. If you don't do this, the animation won't export out with the file.

From there, depending on what your use case is, you use the "autoplay" tag inside the <model-view> element. With that your animation will play/loop on page load.

**Reply by renanmalato** [reply]
Thank you. I could make it works.

I got 2 more questions about it


1. Could you export GBL with 2 objects animations merged? 
Like I have the Animation movement of my object and the Light animation. Blend it or maybe call it in the "animation-name="Light, Object"

<img width="376" alt="Screen Shot 2023-12-24 at 1 34 00 PM" src="https://github.com/google/model-viewer/assets/3617070/185a9288-917b-45fb-b8a3-6bd133752932">

2. Are you able to make Materials animations and move it to the Action Editor?
I mean, animate the Light Emission?

Thanks
